src/main/resources/db/changelog/changes/02-create-user-table.yaml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-table
+      author: nklimovych
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
```suggestion
```

Reply:
I added this constraint based on the information provided in the material from this article: [Database change management tools](https://mate.academy/learn/java-hibernate/java-db-change-management-tools/linear?section=theory&testTaskSlug=java_liquibase_practice&theoryId=2588&videoId=1199)

Reply:
I get it. I have deleted this constraint )